app/src/androidTest/java/com/github/sdp/tarjetakuna/database/localDatabase/LocalDatabaseTest.kt

```diff
@@ -0,0 +1,112 @@
+package com.github.sdp.tarjetakuna.database.localDatabase
+
+import androidx.test.core.app.ActivityScenario
+import androidx.test.core.app.ApplicationProvider
+import androidx.test.ext.junit.runners.AndroidJUnit4
+import com.github.sdp.tarjetakuna.MainActivity
+import com.github.sdp.tarjetakuna.database.DBMagicCard
+import com.github.sdp.tarjetakuna.database.local.AppDatabase
+import com.github.sdp.tarjetakuna.database.local.LocalDatabaseProvider
+import com.github.sdp.tarjetakuna.utils.TemporaryCards.generateCards
+import kotlinx.coroutines.runBlocking
+import kotlinx.coroutines.withTimeout
+import org.junit.After
+import org.junit.Before
+import org.junit.Test
+import org.junit.runner.RunWith
+
+@RunWith(AndroidJUnit4::class)
+class LocalDatabaseTest {
+
+    private lateinit var activityRule: ActivityScenario<MainActivity>

```

Comment:
Do you really need an activity for this test? Why not using only unitTest?

Reply:
I need a context in order to do the tests and it did not work when I tried in the test without UI